.github/workflows/lth-docker.yml

```
@@ -51,10 +51,11 @@ jobs:
         continue-on-error: false # Continue to run the action even if the previous steps fail
         env:
           LIQUIBASE_PRO_LICENSE_KEY: ${{ env.PRO_LICENSE_KEY }} # Set the environment variable for the Liquibase Pro license key
-        run: mvn -ntp -Dtest=${{ matrix.liquibase-support-level }}ExtensionHarnessSuite test # Run the Liquibase test harness at each test level
+          TEST_LEVEL: ${{ matrix.liquibase-support-level }}
+        run: mvn -ntp -Dtest=${TEST_LEVEL}ExtensionHarnessSuite test # Run the Liquibase test harness at each test level
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:1:17: Double quote to prevent globbing and word splitting

(shellcheck)